pkg/resource/nodecadaemon.go

```diff
@@ -74,25 +64,12 @@ spec:
             done
             for d in $(ls /etc/docker/certs.d); do
                 echo $d
-                if [ "${d}" == "${internalRegistryHostname}" ]; then
-                    continue
-                fi
-                if [ "${d}" == "image-registry.openshift-image-registry.svc.cluster.local:5000" ]; then
-                    continue
-                fi
                 dp=$(echo $d | sed  -r 's/(.*):/\1\.\./')
                 reg_conf_path="/tmp/serviceca/${dp}"
                 if [ ! -e "${reg_conf_path}" ]; then
                     rm -rf /etc/docker/certs.d/$d
                 fi
             done
-            if [ -e /tmp/serviceca/service-ca.crt ]; then
-              cp -u /tmp/serviceca/service-ca.crt /etc/docker/certs.d/image-registry.openshift-image-registry.svc.cluster.local:5000
-              cp -u /tmp/serviceca/service-ca.crt /etc/docker/certs.d/${internalRegistryHostname}
-            else 
-              rm /etc/docker/certs.d/image-registry.openshift-image-registry.svc.cluster.local:5000/service-ca.crt
-              rm /etc/docker/certs.d/${internalRegistryHostname}/service-ca.crt
-            fi
```

Comment:
@dmage If I read the PR correctly - we don't need this any more because the ServiceCA ConfigMap is created first, and then the certificate data is copied into the registry's "global" CA keyed on the internal registry hostname.

Reply:
right, but we don't put `image-registry.openshift-image-registry.svc.cluster.local:5000` to this configmap.

Reply:
per @bparees - let's restore this for now. We can revisit in 4.2 when we hopefully are able to drop the port number.